src/main/java/training/supportbank/Main.java

```diff
@@ -1,8 +1,31 @@
 package training.supportbank;
 
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
 public class Main {
+
+    private static final Logger LOGGER = new LogManager();
+
     public static void main(String args[]) {
-        // Your code here!
-        System.out.println("Test!");
+
+        LOGGER.log(Level.FINEST, "testing logging on startup");
+
+        Scanner scanner = new Scanner(System.in);
+        Bank bank = new Bank();
+        FileService fs = new FileService("./Transactions2014.csv");
+        FileService dodgyfs = new FileService("./DodgyTransactions2015.csv");
+        Printer p = new Printer(bank);
+        UserInterface ui = new UserInterface(p);

```

Comment:
You've composed the classes very nicely. Everything has a simple, obvious purpose and they fit together cleanly. 👍